### DISCUSSION TITLE
V3.0's Hello NAS demo can't run

### DISCUSSION THREAD

**Discussion by yyh7773** [microsoft/nni#5707] [Q&A]
'powershell.exe' �����ڲ����ⲿ���Ҳ���ǿ����еĳ���
���������ļ���
'powershell.exe' �����ڲ����ⲿ���Ҳ���ǿ����еĳ���
���������ļ���
'powershell.exe' �����ڲ����ⲿ���Ҳ���ǿ����еĳ���
���������ļ���
'powershell.exe' �����ڲ����ⲿ���Ҳ���ǿ����еĳ���
���������ļ���
'powershell.exe' �����ڲ����ⲿ���Ҳ���ǿ����еĳ���
���������ļ���
[2023-11-06 16:13:47] ERROR: Strategy failed to execute.

**Comment by Baiyun-u-smartAI** [comment]
Maybe the environment is broken, or you could try to use this for test https://github.com/microsoft/nni/blob/36ba04c94e51eaa2a88b9e6e0a4077f3a9f93004/examples/tutorials/hello_nas.py
